source/data_api.html.md

```diff
@@ -4135,6 +4168,73 @@ page[size] <br /><code><a href='#types'>optional</a> <a href='#types'>integer</a
 
 
 
+### Find By Post
+
+Searches the collection, returning a list of matching resources. This `find by post` endpoint is a POST request.
+Parameters must be included in the request body and not as URL parameters.
+
+
+
+#### Invocation
+
+> Example Request
+
+```shell
+curl -XPOST /v1/phone_numbers/find
+```
+
+`POST /v1/phone_numbers/find`
+
+
+> Example Response
+
+```json
+[
+  {
+    "data": [
+      {
+        "id": "c9a72425-f433-4c6c-9d95-4057b25acc2f",
+        "patient_id": "9a75cd83-7247-4d6b-a1dd-00e1aca2219f",
+        "phone_number": "+15555555555",
+        "phone_number_type": "landline",
+        "friendly_name": "main number",
+        "verified": false,
+        "opted_in_to_sms": true,
+        "opted_in_to_call_recording": false,
+        "opted_in_to_voicemail": false,
+        "opted_in_to_phone": true,
+        "automatic_recipient": false,
+        "updated_at": "2018-09-12T01:27:32.123172+00:00",
+        "created_at": "2018-09-12T01:27:32.123301+00:00"
+      }
+    ],
+    "meta": {
+      "current": {
```

Comment:
I thought we still provided an href, just without url params--am I confused?

Reply:
hmmm not sure. this is a bug that i will track separate from this docs release